flink-staging/flink-hcatalog/pom.xml

```diff
@@ -34,17 +34,64 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<repositories>
+		<repository>
+			<id>cloudera</id>
+			<url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
+		</repository>
+	</repositories>
+
```

Comment:
We should not depend on vendor specific repositories / versions in the normal builds.
In the parent pom, there is a profile to enable vendor repositories.